pkgs/top-level/all-packages.nix

```diff
@@ -273,6 +273,8 @@ with pkgs;
 
   altserver-linux = callPackage ../tools/misc/altserver-linux { };
 
+  arxiv-latex-cleaner = callPackage ../tools/misc/arxiv-latex-cleaner { };
```

Comment:
Please move the package file to `pkgs/by-name/ar/arxiv-latex-cleaner/package.nix` and remove it from `all-packages.nix`.
See here for more information: https://github.com/NixOS/rfcs/blob/master/rfcs/0140-simple-package-paths.md

Reply:
That's much cleaner. Updated. Thanks